authors/apps/authentication/views.py

```diff
@@ -11,11 +11,14 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from sendgrid.helpers.mail import *
+from .social_login.common import BaseAtrributes
 
 from .backends import JWTAuthentication
 from .renderers import UserJSONRenderer
-from .serializers import (LoginSerializer, RegistrationSerializer,
-                          UserSerializer)
+from .serializers import (
```

Comment:
See previous comment.

Reply:
Thank you @lym , am going to work it

Reply:
@lym, I worked on the comments you suggested